crates/trigger/src/cli/stdio.rs

```diff
@@ -86,6 +95,24 @@ impl StdioLoggingExecutorHooks {
             _ => Ok(()),
         }
     }
+
+    fn truncate_log_files(&self, log_dir: &Option<PathBuf>) {
```

Comment:
This truncates all log-like files in the directory - as discussed before, this potentially runs into a problem with multi-trigger if one trigger process emits a log while the other is still starting.

Maybe that's not a consideration, we probably have race conditions in multi-trigger already (like KV init), but I'd definitely seek a second opinion...

Reply:
We have a race condition in a _single_ trigger; each instance opens a separate handle to the log file and writes to it without any locking. In the context of `spin up` this behavior seems fine to me.